python/pyarrow/_flight.pyx

```diff
@@ -746,6 +759,24 @@ cdef class FlightEndpoint(_Weakrefable):
         return [Location.wrap(location)
                 for location in self.endpoint.locations]
 
+    @property
+    def expiration_time(self):
```

Comment:
Can you add docstrings for these new properties?

Reply:
sure